spikes/input spike/InputHandler.cpp

```diff
@@ -0,0 +1,269 @@
+#include "stdafx.h"
+#include "InputHandler.h"
+using namespace std;
+//this class handles the inputs of the controller to be used outside this class cleanly
+
+//if (inputHandler.getReleasedButton() == InputHandler::BUTTON::A)  <-- use the method to get the key pressed in your class outside this one
+//{}
+InputHandler::InputHandler()
+{
+	wasButtonAPressed = false;
+	wasButtonBPressed = false;
+	wasButtonXPressed = false;
```

Comment:
yes, 
InputHandler::InputHandler() : wasButtonAPressed(false), wasButtonBPressed(false)
{

do this with all constructor values you can, only move it down if its complex.